parsers/EIA.py

```diff
@@ -32,16 +36,28 @@
 
 def fetch_consumption_forecast(zone_key, session=None, target_datetime=None,
                                logger=None):
+    return _fetch_production_or_consumption(
+        zone_key, DAY_AHEAD[zone_key], session=session,
+        target_datetime=target_datetime, logger=logger)
+
+
+def fetch_production(zone_key, session=None, target_datetime=None,
+                     logger=None):
+    return _fetch_production_or_consumption(
+        zone_key, PRODUCTION[zone_key], session=session,
+        target_datetime=target_datetime, logger=logger)
+
 
-    series_id = DAY_AHEAD[zone_key]
+def _fetch_production_or_consumption(zone_key, series_id, session=None,
```

Comment:
how do we know if it's a production or consumption?

Reply:
good question. the series id, specifically `DF` for consumption forecast vs `NG` for production. i've added a docstring.